lib/oauth2.js

```diff
@@ -115,15 +126,33 @@ Strategy.prototype.userProfile = function(accessToken, done) {
             return true
           }
         })
-      } 
+      }
+      if(!profile.email && json.email) {
```

Comment:
Inconsistent spacing.

```suggestion
      if (!profile.email && json.email) {
```